sdcm/cluster.py

```diff
@@ -1873,10 +1875,73 @@ def run_nodetool(self, sub_cmd, args="", options="", ignore_status=False):
         """
         cmd = self._gen_nodetool_cmd(sub_cmd, args, options)
         result = self.remoter.run(cmd, ignore_status=ignore_status)
-        self.log.debug("Command '%s' duration -> %s s", result.command,
-                       result.duration)
+        self.log.debug("Command '%s' duration -> %s s" % (result.command, result.duration))
+
         return result
 
+    def check_node_health(self, target_node=None):
+        self.check_nodes_status()
+        self.check_schema_version()
+
+    def check_nodes_status(self, target_node=None):
```

Comment:
`check_nodes_status` -> `check_node_status`

Reply:
This function checks status of all nodes, not one

Reply:
Changed

Reply:
this is a method on the node class, so it's working, and being called on each node in `check_cluster_health()`, so bentsi has a valid point here.